CI/travis.linux.after_success.sh

```diff
@@ -114,9 +112,21 @@ if { [ "${DEPLOY}" = "deploy" ]; } ||
     fi
 
     if [ "${public_test_build}" == "true" ]; then
-      echo "=== Uploading public test build to make.mudlet.org ==="
-      DEPLOY_URL=$(wget --method PUT --body-file="Mudlet-${VERSION}${MUDLET_VERSION_BUILD}-linux-x64.AppImage.tar" \
-                     "https://make.mudlet.org/snapshots/Mudlet-${VERSION}${MUDLET_VERSION_BUILD}-linux-x64.AppImage.tar" -O - -q)
+      # if [ -n "$TRAVIS_REPO_SLUG" ]; then
+        echo "=== Uploading public test build to make.mudlet.org ==="
+        DEPLOY_URL=$(wget --method PUT --body-file="Mudlet-${VERSION}${MUDLET_VERSION_BUILD}-linux-x64.AppImage.tar" \
+                      "https://make.mudlet.org/snapshots/Mudlet-${VERSION}${MUDLET_VERSION_BUILD}-linux-x64.AppImage.tar" -O - -q)
+      else
+      #   echo "=== ... later, via Github ==="
```

Comment:
Like wot codefactor sez: you must have a non-comment line here - even if it is the suggested `true` - which of course "*does nothing, but does it successfully*"!